pages/nrfnano.vue

```diff
@@ -322,9 +322,14 @@ export default {
     const products = (await $axios.get(`/api/stripe/products`)).data
     const product = products.find((p) => p.codename === 'badgerd_nrfnano')
     const shippingRates = (await $axios.get(`/api/shippingrates`)).data
-    const countries = []
-    for (const countryCode in shippingRates) {
-      countries.push({ code: countryCode, name: shippingRates[countryCode].name })
+    let countries = []
+
+    if (Object.prototype.hasOwnProperty.call(shippingRates, 'countries')) {
+      countries = shippingRates.countries
+    } else {
+      for (const countryCode in shippingRates) {
+        countries.push({ code: countryCode, name: shippingRates[countryCode].name })
+      }
```

Comment:
I understand that shippingrates now have countries property,  if not we are using same way to fill countries list.  LGTM!